Src/WindowsServer/WindowsServer.Shared/FirstChanceExceptionStatisticsTelemetryModule.cs

```diff
@@ -389,6 +389,9 @@ private void SendException(string operationName, string problemId, ExceptionTele
                     exceptionTelemetry.ProblemId = problemId;
                 }
 
+                // this property allows to differenciate examples from regular exceptions tracked using TrackException
+                exceptionTelemetry.Properties.Add("_MS.ProcessedByMetricExtractors", "(Name: Exceptions, Ver: 1.0)");
```

Comment:
It is the same as used by metrics aggregator from Greg. It is design to:
1. Be human readable
2. have a prefix indicating Microsoft ownership
3. tell the name and version of the extractor

The plan is to include it into the context as a well-known field eventually.

Do you have a better name in mind?